awx/main/models/inventory.py

```diff
@@ -1364,9 +1364,9 @@ def _get_unified_job_class(cls):
 
     @classmethod
     def _get_unified_job_field_names(cls):
```

Comment:
This change makes sense to me, but I'd prefer we use `.union()` instead of the `|` operator.